documentation/azure-powershell-modules.md

```diff
@@ -42,7 +42,7 @@ Custom Resource Providers             | `Az.CustomProviders`
 Data Box                              | `Az.DataBox`                            | [![DataBox]][DataBoxGallery]
 Data Box Edge (Stack Edge)            | `Az.DataBoxEdge`                        | [![DataBoxEdge]][DataBoxEdgeGallery]
 Databricks                            | `Az.Databricks`                         | [![Databricks]][DatabricksGallery]
-DataDog                               | `Az.DataDog`                            | [![DataDog]][DataDogGallery]
```

Comment:
Three lines requires to change in this file. Please preview the mark down file

Reply:
Resolved  in https://github.com/Azure/azure-powershell/pull/15303